web-api/src/trialSessions/saveCalendarNoteLambda.js

```diff
@@ -1,15 +1,20 @@
 const { genericHandler } = require('../genericHandler');
 
 /**
- * creates a new case record from pre-existing case data
+ * used for saving a case's calendar note for a trial session
  *
  * @param {object} event the AWS event object
  * @returns {Promise<*|undefined>} the api gateway response object containing the statusCode, body, and headers
  */
-exports.migrateCaseLambda = event =>
+exports.saveCalendarNoteLambda = event =>
   genericHandler(event, async ({ applicationContext }) => {
-    return await applicationContext.getMigrations().migrateCaseInteractor({
+    const lambdaArguments = {
+      ...event.pathParameters,
+      ...JSON.parse(event.body),
+    };
+
+    return await applicationContext.getUseCases().saveCalendarNoteInteractor({
       applicationContext,
-      caseMetadata: JSON.parse(event.body),
+      ...lambdaArguments,
```

Comment:
This just doesn't feel right. Can we please pass in what `saveCalendarNoteInteractor` needs.

Reply:
... have created Trello cards to help develop a better pattern.

Reply:
https://trello.com/c/3gH4hRRi